modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/info/item/provider/test/DDMFormValuesInfoFieldValuesProviderTest.java

```diff
@@ -105,6 +122,44 @@ public void testGetInfoFieldValuesCheckboxMultipleDDMFormFieldType()
 				).build()));
 	}
 
+	@Test
+	public void testGetInfoFieldValuesNestedStructure() throws Exception {
+		DDMStructure ddmStructure = _addDDMStructure(
+			_group, "test-structured-content-nested-structure.json");
+
+		long classNameId = ClassNameLocalServiceUtil.getClassNameId(
+			JournalArticle.class);
+
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext(
+				_group.getGroupId(), TestPropsValues.getUserId());
+
+		JournalFolder journalFolder = _addJournalFolder(
+			ddmStructure.getStructureId(), serviceContext);
+
+		DDMTemplate ddmTemplate = DDMTemplateTestUtil.addTemplate(
+			_group.getGroupId(), ddmStructure.getStructureId(), classNameId);
+
+		String content = _read("ddm-structure-content-nested.xsd");
```

Comment:
Could you rename this file name to `ddm-structure-fieldset-field.sxd` ? This is to be in line with the names of the other files.
What do you think?

Reply:
Sure!